crates/l2/contracts/src/l1/CommonBridge.sol

```diff
@@ -49,6 +51,14 @@ contract CommonBridge is
     /// @dev It's used to validate withdrawals
     address public constant L2_BRIDGE_ADDRESS = address(0xffff);
 
+    /// @notice How much of each L1 token was deposited to each L2 token.
+    /// @dev Stored as L1 -> L2 -> amount
+    /// @dev Prevents L2 tokens from faking their L1 address and stealing tokens
+    mapping (address => mapping (address => uint256)) public depositsERC20;
+
+    /// @notice Token address used to represent ETH
+    address public constant ETH_TOKEN =  0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
```

Comment:
nit: maybe this should be all-downcase?

Reply:
We could also shorten it to `0xeeee`

Reply:
It's the checksummed version ([ref](https://eips.ethereum.org/EIPS/eip-55))